provisioning_templates/provision/preseed_default.erb

```diff
@@ -122,7 +123,7 @@ d-i apt-setup/local<%= repos %>/key string http://keyserver.ubuntu.com/pks/looku
 tasksel tasksel/first multiselect minimal, ssh-server, openssh-server
 
 # Install some base packages
-d-i pkgsel/include string <%= salt_package %> lsb-release
+d-i pkgsel/include string <%= salt_package %> <%= ansible_enabled ? 'python' : '' %> lsb-release <%= host_param('additional-packages') %>
```

Comment:
Will there be an issue with python vs python3? Would it be an idea to use list?
```ruby
additional_packages = ['lsb-release']
additional_packages << host_param('additional-packages')
additional_packages << 'salt-minion' if salt_enabled
additional_packages << 'python' if ansible_enabled
```

Not sure if our ERB sandbox allows this.

Reply:
It looks cleaner. ;)

Reply:
There is no problem having both python 2 and 3 installed simultaneously.